### ISSUE TITLE
anti-aliasing

### ISSUE THREAD

**Issue by Unlingius** [carla-simulator/carla#1133]
Hi!
I observe, that anti-aliasing function is turned on in main window, but not in client.
![image](https://user-images.githubusercontent.com/40238431/51109419-3cc62600-1806-11e9-8df9-677122852df2.png)
is there any way to activate it? Thanks.

**Comment by manishthani** [comment]
Hi @Unlingius
 
Right now this functionality is not available due to some issues related with Unreal engine built on Linux that makes the server send the frame to the client **without** some post processing effects, where one of them is anti-aliasing. 
As an alternative, it could be possible to use [**FXAA**](https://developer.download.nvidia.com/assets/gamedev/files/sdk/11/FXAA_WhitePaper.pdf) post processing effect on the received frame from the server which is **already** implemented by Nvidia in their [Github repository](https://github.com/NVIDIAGameWorks).

Hope this helps!
Manish

**Comment by Unlingius** [comment]
@manishthani  Thank you for reply. Can you recommend any easy-to-use tutorial how to try it on given frame? I even don't need realtime speed.

**Comment by Unlingius** [comment]
I tried SMAA morphological aintialising from https://github.com/iRi-E/smaa-cpp and got result:
![image](https://user-images.githubusercontent.com/40238431/51182201-78ccba00-18de-11e9-9dec-076a30e99940.png)
It works good with "stairs", but not with gaps. It seems, any postprocessing methods are not enough in this case. I found, that maybe useful follows methods: tesselation shaders, conservative rasterization, SSAA, MSAA. They are not postprocessing. Is that possible to activate it in unreal settings?
